DataProducts/inc/CrvStatus.hh

```diff
@@ -0,0 +1,46 @@
+#ifndef RecoDataProducts_CrvStatus_hh
+#define RecoDataProducts_CrvStatus_hh
+//
+//
+// Contact person Simon Corrodi
+//
+
+#include <array>
+#include <vector>
+
+namespace mu2e
+{
+  class CrvStatus
```

Comment:
Is this a per-event product?  Per Run?  Per SubRun?

Reply:
It's on the event level. We'll get multiple per events - one per CRV ROC (17). I don't expect them to be used in an analysis chain, these are really for debugging.